lib/workers/repository/index.js

```diff
@@ -12,7 +12,7 @@ const { ensureMasterIssue } = require('./master-issue');
 export { renovateRepository };
 
 // istanbul ignore next
-async function renovateRepository(repoConfig) {
+async function renovateRepository(repoConfig, prsAlreadyCreated) {
```

Comment:
Please work out a solution that doesn't require a change to this function parameters